build.gradle

```diff
@@ -2175,7 +2176,7 @@ project(':tools') {
     implementation libs.jacksonDataformatCsv
     implementation libs.jacksonJDK8Datatypes
     implementation libs.slf4jApi
-    implementation libs.slf4jlog4j
```

Comment:
Could we replace slf4jlog4j with slf4j-reload4j evrywhere, and remove libs.slf4jlog4j altogether?

Reply:
Done. 
- replaced slf4jlog4j with slf4jReload4j
- and removed libs.slf4jlog4j